app/templates/logout.html

```diff
@@ -1,4 +1,4 @@
 {% extends "base.html" %}
 {% block content %}
-<h3>Will see soon</h3>
+<div class="alert alert-success logout" role="alert">You've been logged out successfully.</div>
```

Comment:
OK.
Regarding container class, styles are probably cached on your machine, it should be displayed differently.